python/python/glide/async_commands/core.py

```diff
@@ -2703,6 +2703,54 @@ async def xrange(
             await self._execute_command(RequestType.XRange, args),
         )
 
+    async def xrevrange(
+        self,
+        key: str,
+        end: StreamRangeBound,
+        start: StreamRangeBound,
+        count: Optional[int] = None,
+    ) -> Optional[Mapping[str, List[List[str]]]]:
+        """
+        Returns stream entries matching a given range of IDs in reverse order. Equivalent to `XRANGE` but returns the
+        entries in reverse order.
+
+        See https://valkey.io/commands/xrevrange for more details.
+
+        Args:
+            key (str): The key of the stream.
+            end (StreamRangeBound): The ending stream ID bound for the range.
+                - Use `IdBound` to specify a stream ID.
+                - Use `ExclusiveIdBound` to specify an exclusive bounded stream ID.
+                - Use `MaxId` to end with the maximum available ID.
+            start (StreamRangeBound): The starting stream ID bound for the range.
+                - Use `IdBound` to specify a stream ID.
+                - Use `ExclusiveIdBound` to specify an exclusive bounded stream ID.
+                - Use `MinId` to start with the minimum available ID.
+            count (Optional[int]): An optional argument specifying the maximum count of stream entries to return.
+                If `count` is not provided, all stream entries in the range will be returned.
+
+        Returns:
+            Optional[Mapping[str, List[List[str]]]]: A mapping of stream IDs to stream entry data, where entry data is a
```

Comment:
Why it is optional? Should we tell when command returns None?

Reply:
the command returns None if you provide a count value < 1. which is a bit strange because calling it against an empty stream or non existing key returns an empty dict. Calling when no streams are in the range also returns an empty dict. I'm not sure if we should specify that it returns None in this case since its a bit of a strange use case

Reply:
...I don't know that it every returns null

Reply:
I take that back.  It returns null if the input values don't make sense.